### PR TITLE
New type definition for jsspec

### PR DESCRIPTION
**Pull request by HookyQR**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/jsspec__jsspec/tslint.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (jessetrinity) [inline types/jsspec__jsspec/tslint.json]: Hi @HookyQR , thanks for your contribution. We prefer not to have rules turned off for new definitions unless absolutely necessary.

**Review by jessetrinity — CHANGES_REQUESTED** [review]

File: types/jsspec__jsspec/tslint.json (hunk 1 above)
Comment (HookyQR) [reply types/jsspec__jsspec/tslint.json]: On it. 👍

Comment (HookyQR) [reply types/jsspec__jsspec/tslint.json]: Back to you.

**Comment by jessetrinity** [comment]
Sorry, I'm not familiar with this library and I don't see any usage examples. Is it a global or modular library (would you expect to have to import JSSpec)?

**Comment by HookyQR** [comment]
It's global. It's a test runner. You would not import this. If it was imported, I would have been able to put the types in the library itself, but alas, it needs it's own devDependancy for the types.

**Comment by jessetrinity** [comment]
Awesome, congrats on your first contribution to DefinitelyTyped!

**Pull request merged**